src/Presentation/Camera/Fetching/Models/TPCamera.cs

```diff
@@ -0,0 +1,16 @@
+namespace Presentation.Camera.Fetching.Models
```

Comment:
Should we rename the `Models` into `Types`? It's the presentation layer. We don't have models here.

Reply:
Let's also make sure we rename folders and adjust namespaces.

Reply:
Done